rpcs3/Emu/Cell/lv2/sys_storage.cpp

```diff
@@ -1,37 +1,85 @@
 #include "stdafx.h"
+#include "Emu/System.h"
+#include "Emu/Memory/vm.h"
+#include "Emu/IdManager.h"
 
 #include "Emu/Cell/ErrorCodes.h"
+#include "Emu/Cell/lv2/sys_event.h"
 
 #include "sys_storage.h"
 
-
 LOG_CHANNEL(sys_storage);
 
+u32 asyncequeue;
+
 error_code sys_storage_open(u64 device, u64 mode, vm::ptr<u32> fd, u64 flags)
 {
 	sys_storage.todo("sys_storage_open(device=0x%x, mode=0x%x, fd=*0x%x, flags=0x%x)", device, mode, fd, flags);
 
-	return CELL_OK;
+	if (device == 0)
+	{
+		return CELL_ENOENT;
+	}
+
+	u64 storage_id = device & 0xFFFFF00FFFFFFFF;
+	fs::file file;
```

Comment:
What is the meaning empty file here?

Reply:
file is const as well, it remains empty yet other places suggest it should contain somthing.

Reply:
I believe there used to be code to load the nor region file in there.., it doesn't seem to effect anything however, and im not sure what it should look like myself.